mlflow/store/file_store.py

```diff
@@ -407,12 +407,19 @@ def _get_run_files(self, run_uuid, resource_type):
     @staticmethod
     def _get_metric_from_file(parent_path, metric_name):
         _validate_metric_name(metric_name)
-        metric_data = read_file_lines(parent_path, metric_name)
+        metric_data = [
+           line.strip().split(" ") for line in read_file_lines(parent_path, metric_name)
```

Comment:
Minor suggestion:

```suggestion
           line.split() for line in read_file_lines(parent_path, metric_name)
```

Using `.split()` without an argument is tolerant to multiple space and tabs etc., and so is a less likely to fail if the file contents change for some reason. It also disregards any leading or trailing whitespace and so makes the `.strip()` call redundant.